.github/workflows/e2e-test.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Install dependencies
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: 1.20
```

Comment:
This should be `"1.20"` instead of `1.20` which will be recognized as 1.2. I'm sure this misleading lot's of people. Please fix this and we are finally done.